source/index.html.md

```diff
@@ -499,6 +500,7 @@ curl "https://www.filmmakers.eu/api/v1/actor_profiles/123" \
         "name": "Schneeflöckchen",
         "year_from": 2019,
         "year_to": 2020,
+        "in_development": false,
```

Comment:
Can we add vita_entries to the Response fields above, and explain the new value? Also, let's add `vita_entries` (array) and explain the sorting order in which these are emitted (I've had multiple questions re this over the last months)

e.g. 


**vita_entries[]**: An array of credits. Each entry contains structured data related to a specific credit. The entries in this array are sorted in XXXXXXXXXX order.

**vita_entries[].in_development**: indicates whether the title is still in development, i.e. not yet completed/released.